blockchain/vm/gas_table.go

```diff
@@ -338,9 +216,9 @@ func gasExp(gt params.GasTable, evm *EVM, contract *Contract, stack *Stack, mem
 	return gas, nil
 }
 
-func gasCall(gt params.GasTable, evm *EVM, contract *Contract, stack *Stack, mem *Memory, memorySize uint64) (uint64, error) {
+func gasCall(evm *EVM, contract *Contract, stack *Stack, mem *Memory, memorySize uint64) (uint64, error) {
 	var (
-		gas            = gt.Calls
+		gas            uint64
```

Comment:
The original default value was 700 and now it is chnaged to 0. Is there something changed is gas value?

Reply:
As we talked offline, it looks like this code is only used for checking enough value in wallet. It is just for estimation and 700 gas was added for surplus.
The actual gas cost is calculated in `operation.execute`.